README.rst

```diff
@@ -44,7 +44,8 @@ Quick start
 
    This is tested and known to work on OSX and Windows. On Linux, there is a
    permissions issue if (numerical) user id on the host system and in the
-   container deviate.
+   container deviate. Moreover, in Linux, the files generated would be owned by root,
+   and thus could cause issues with further file manipulation (such as deleting files).
```

Comment:
What is the worst-case scenario here?

Reply:
```console
❯ cp ../examples/he/inp .
❯ docker run --rm -ti -v $PWD:/io fangohr/octopus octopus "$@"

    <><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
                                ___
                             .-'   `'.
                            /         \
                            |         ;
                            |         |           ___.--,
                   _.._     |0) ~ (0) |    _.---'`__.-( (_.
            __.--'`_.. '.__.\    '--. \_.-' ,.--'`     `""`
           ( ,.--'`   ',__ /./;   ;, '.__.'`    __
           _`) )  .---.__.' / |   |\   \__..--""  """--.,_
          `---' .'.''-._.-'`_./  /\ '.  \ _.-~~~````~~~-._`-.__.'
                | |  .' _.-' |  |  \  \  '.               `~---`
                 \ \/ .'     \  \   '. '-._)
                  \/ /        \  \    `=.__`~-.
             jgs  / /\         `) )    / / `"".`\
            , _.-'.'\ \        / /    ( (     / /
             `--~`   ) )    .-'.'      '.'.  | (
                    (/`    ( (`          ) )  '-;
                     `      '-;         (-'

[..cntd..]
❯ ls
 exec   inp   output_iter   restart   static
❯ rm -rf restart
rm: cannot remove 'restart/gs/indices.obf': Permission denied
rm: cannot remove 'restart/gs/grid': Permission denied
rm: cannot remove 'restart/gs/states': Permission denied
rm: cannot remove 'restart/gs/wfns': Permission denied
rm: cannot remove 'restart/gs/occs': Permission denied
rm: cannot remove 'restart/gs/0000000001.obf': Permission denied
rm: cannot remove 'restart/gs/density': Permission denied
rm: cannot remove 'restart/gs/density.obf': Permission denied
❯ ls -lah restart
Permissions Size User Date Modified Name
drwxr-xr-x@    - root  6 Feb 16:18   gs
❯ touch exec/i_want_to_put_some_data
touch: cannot touch 'exec/i_want_to_put_some_data': Permission denied
❯ touch exec/i_want_to_log_something
touch: cannot touch 'exec/i_want_to_log_something': Permission denied
❯ touch exec/anything_please
touch: cannot touch 'exec/anything_please': Permission denied
```

Reply:
That's true. But to be able to run the docker container, you need to have root/sudo rights (or you have added yourself to the `docker` group as sudo/root before). 

I hesitate to have 4 lines of text that talk about problems which in reality hardly anybody should need (because we don't expect Linux users to use this. And even if they do, there was a warning about file permissions. And they have root rights anyway because otherwise they cannot run the docker container). And most of the readers (Windows and macOS users) would not be able to make sense of those 4 lines either.

It would be better, of course, if we can make this work nicer on Linux, but perhaps not highest priority at the moment. I'll close this for now.